AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -1121,7 +1121,13 @@ - (void)notifyWithUserConfirmation:(MSUserConfirmation)userConfirmation {
      */
     [MS_USER_DEFAULTS setObject:[[NSNumber alloc] initWithBool:YES] forKey:kMSUserConfirmationKey];
   }
-
+  
+  // Check if there is no handler set and unprocessedReports are not initialized as NSMutableArray (Init occurs in correct call sequence).
+  if ([MSCrashes sharedInstance].userConfirmationHandler==nil && !self.unprocessedReports) {
```

Comment:
Is `[MSCrashes sharedInstance].userConfirmationHandler == nil` a necessary check?
`notifyWithUserConfirmation` is also called by Crashes module, it might not be needed.

Reply:
Why is the check in the middle of the method? If it is a condition to stop executing the method, we should stop and return as early as possible.